### ISSUE TITLE
Please help me understanding I2C protocol on ATTiny85

### ISSUE THREAD

**Issue by Banfotech** [Banfotech/ATTiny85-I2C#1]
Hello Everyone!
I'm a completly newbie on coding so maybe this could be a stupid question, but at the moment I haven't find a solution.

I'm approaching for the very first time the I2C protocol, I'm using an ATTiny 85 board with the TinyWire library.

I've set up the communication between an Arduino Nano, running the I2C scanner, and they communicate.

Now I would like to do the "Hello World" of electronics, turning on a led, sending a code from the Master (the Nano) to the Slave (the ATTiny85).

The TinyWire libraries have some example like Master_Reciever, Master_Sender ecc...
But i'm still confused. 

**Summing up the goal is:**

Send a caracter, a number, what do you prefere from the Master to the Slave, once the Slave recieve that specific data, turn on or off the led.

Please, there is someone who would like to help me?

Many thanks for your time!
